datagrepper/app.py

```diff
@@ -333,13 +335,13 @@ def raw():
     if chrome not in ['true', 'false']:
         raise ValueError("chrome should be either 'true' or 'false'")
 
-    if contains and datetime.fromtimestamp(start) < (datetime.utcnow() - timedelta(weeks=4*8)):
-        raise flask.BadRequest('When using contains, specify a start at most '
-                               'eight months into the past')
+    if contains and datetime.fromtimestamp(start or 0) < (datetime.utcnow() - timedelta(weeks=4*8)):
```

Comment:
This isn't technically eight months :stuck_out_tongue:

Reply:
"eight months"